cmd/runtimecfg/node-ip.go

```diff
@@ -206,10 +208,15 @@ func checkAddressUsable(chosen []net.IP) (err error) {
 }
 
 func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType string) (chosen []net.IP, matchesVips bool, err error) {
+	// timerLoop will hold a time in Seconds to be used with time.Sleep() before going
+	// for the next loop interation.
+	timerLoop := 1
+
 	// Enable debug logging in utils package
 	utils.SetDebugLogLevel()
 	ipFilterFunc := utils.ValidNodeAddress
 	for {
+		timerLoop = timerLoop + addSecondsToSuitableIPsLoop
```

Comment:
Oh, I didn't catch this the first time, but is there a reason you're adding instead of multiplying here? That will make it take 150 iterations to get to a 5 minute delay

Reply:
good one.